1. 전처리 파일은 main파일이나 재실행시 오류가 이상하게 간헐적으로 발견돼어서 main1에서 main2의 코드부분만 재실행하면 원활히 진행히 가능합니다.
2. model_traning.py 코드가 LMS에서 안내한 autoint.py를 평가항목에 따라 수정한 파일입니다.
3. model_traning.py내에 2를 붙여놓은건 여러 검증을 하는데 파일명이 똑같아 헷갈리지 않기위해 2를 붙여 최종본으로 완성하였습니다.
4. model_traning.py를 실행시 로컬에서 다음과 같은 평가 결과값을 얻었습니다.
![image](https://github.com/user-attachments/assets/68a72445-2d6c-410a-bfef-0afc86d26825)
5. 최종적으로 show_st.py를 실행히 다음과 같은 화면이 확인 됩니다.
![image](https://github.com/user-attachments/assets/20911b21-78f4-4ec6-a37f-f0aeab4d49e5)
6. 영상은 다음과 같이 확인되어졌습니다.
https://github.com/user-attachments/assets/e17773ba-7478-4f0c-bf48-b27ce7919c72

